docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -330,7 +330,7 @@ As of Sauce Connect Proxy version 4.7.x, the metrics server is disabled by defau
 --metrics-address 1.2.3.4:80 # listens on 1.2.3.4 port 80
 ```
 
-### Viewing Performance Metrics
+### Viewing Sauce Connect Proxy Metrics
```

Comment:
pls revert

Reply:
Why revert? These are not performance metrics